Doc/source/time.rst

```diff
@@ -1,27 +1,154 @@
+#################################################################################
+spacepy.time - Time conversion, manipulation and implementation of Ticktock class
+#################################################################################
 
+Contents
+--------
 
-#########################################################################
-time - Time conversion, manipulation and implementation of Ticktock class
-#########################################################################
+.. contents::
+   :depth: 2
+   :local:
 
-.. automodule:: spacepy.time
+See also the `full API documentation <spacepy.time>`.
 
-.. rubric:: Classes
+Notes
```

Comment:
Again don't really need a header here.

Reply:
I did this just for the table of contents, I just called it Notes because I didn't know what else to title it.

Reply:
The paragraph right under the ToC doesn't need an entry in the ToC. It's okay.

Reply:
Resolved